server/cluster/cluster.go

```diff
@@ -492,7 +491,7 @@ func (c *RaftCluster) HandleStoreHeartbeat(stats *pdpb.StoreStats) error {
 	storeID := stats.GetStoreId()
 	store := c.GetStore(storeID)
 	if store == nil {
-		return core.NewStoreNotFoundErr(storeID)
+		return errors.Errorf("store %v not found", storeID)
```

Comment:
why use `Errorf` rather than errorCode

Reply:
I don't want to change the return value for gRPC interface, so I just keep the original output.